packages/react-router/modules/__tests__/Route-test.js

```diff
@@ -486,3 +486,36 @@ describe("A <Route>", () => {
     });
   });
 });
+
+describe("A partial <Route>", () => {
```

Comment:
I'd probably add a test for `<Route path={["uncle", "aunt"]} /> `

Reply:
Right, haven't thought about this. Anyway, do you know why the `path` property in `<NavLink />` gets escaped? Doesn't make any sense to me but the tests break after my changes.

Reply:
As far as I can tell, it's because the "to" prop is used both for the `to` prop of the Link and for matching against the current location.

https://github.com/ReactTraining/react-router/commit/c3d453ee4eba703bf606a77f61ce08eac8a64d0a 

#5584

Reply:
So, the question is, why is this not part of `matchPath`?